apps/web/src/components/Profile/components/DiversityEquityInclusion/DiversityEquityInclusion.tsx

```diff
@@ -20,7 +20,7 @@ const DiversityEquityInclusion = ({
   onUpdate,
   isUpdating,
   pool,
-}: SectionProps) => {
+}: SectionProps & { pool: Maybe<Pick<Pool, "publishingGroup">> }) => {
```

Comment:
Wonder if it is worth using a generic here since we seem to repeating ourselves alot :thinking: 

```ts
type SectionProps<P> = {
   //...
   pool?: Maybe<P>;
}

// Usage in props
SectionProps<Pick<Pool, "publishingGroup">>
```

Reply:
Good idea, I didn't consider anything like that. Added in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11015/commits/10507c40dc2b9c79b940935801d28ed25d83f2c0